src/main/java/io/cdap/plugin/servicenow/connector/ServiceNowConnectorConfig.java

```diff
@@ -172,17 +231,23 @@ public Double convertToDoubleValue(Object fieldValue) {
     if (fieldValue instanceof String && Strings.isNullOrEmpty(String.valueOf(fieldValue))) {
       return null;
     }
-
-    return Double.parseDouble(String.valueOf(fieldValue));
+    try {
+      return NumberFormat.getNumberInstance(java.util.Locale.US).parse(String.valueOf(fieldValue)).doubleValue();
```

Comment:
I'm confused by all these convert methods. They all do `String.valueOf(fieldValue)`. What are the actual classes in the Map? Are they always strings? If so, why is it being passed as `Map<String, Object>` instead of `Map<String, String>`? If these values are other classes, why aren't they being cast to the proper class instead of being converted to a string and then converted back?

Reply:
Yes, they are always strings. 
Removed `String.valueOf(fieldValue)` and Replaced `Map<String, Object>` with `Map<String, String>` a963c57e661447a3f1a6e970cdabc6f76964b313

Reply:
corrected. Replaced with `String` arg and removed `String.valueOf()` f58c78717ea204f30540f24a0352ce21ef7ed47b